src/VisualStudio/Core/Def/ColorSchemes/ColorSchemeApplier.ClassificationVerifier.cs

```diff
@@ -107,40 +105,43 @@ public bool AreForegroundColorsCustomized(ColorSchemeName schemeName, Guid theme
                     return false;
                 }
 
-                var coreThemeColors = (themeId == KnownColorThemes.Dark)
+                // Ensure we are initialized
+                await TaskScheduler.Default;
+                _fontAndColorStorage ??= await _asyncServiceProvider.GetServiceAsync<SVsFontAndColorStorage, IVsFontAndColorStorage>().ConfigureAwait(false);
```

Comment:
Want to avoid the switch to the thread pool if we already have this filled out?
```suggestion
               if (_fontAndColorStorage is null)
               {
                    await TaskScheduler.Default;
                    _fontAndColorStorage = await _asyncServiceProvider.GetServiceAsync<SVsFontAndColorStorage, IVsFontAndColorStorage>().ConfigureAwait(false);
                }
```

It's not as cute but might help?

Reply:
seems reasonable to me.